bull-common/src/test/java/com/hotels/transformer/utils/ReflectionUtilsTest.java

```diff
@@ -153,6 +154,21 @@ public void testGetFieldValueWithAGivenFieldWorksAsExpected() throws Exception {
         };
     }
 
+    /**
+     * Tests that the method {@code getFieldValue} catches InvalidBeanException.
```

Comment:
A question: do you think it's possible to merge this method with: `testGetFieldValueWithAGivenFieldWorksAsExpected` using a data provider?

Reply:
I personally think it's not a very good idea to merge these, since in one we are testing that the method works as expected when there are no exceptions, and in the other we are explicitly throwing an unchecked exception inside a nested method to test its behaviour.

Reply:
agreed